python/lsst/ts/mtaos/mtaos.py

```diff
@@ -1006,6 +1020,46 @@ async def run_closed_loop(self):
                         self.log.info(f"Visit {visit_id} already processed, skipping.")
                         continue
 
+                    filter_label, elevation = await self.model.get_image_info(
+                        visit_id,
+                        self.camera_name,
+                    )
+
+                    rotation_angle = float(
+                        np.mean(np.array(self.image_rotator[image_in_oods.obsid]))
+                    )
+
+                    # Since the current rotator and elevation positions
+                    # are being updated in the background, we need to
+                    # create a local copy and assume it's static
+                    # in the following if statement.
+                    current_rotator_position = self.current_rotator_position
+                    current_elevation_position = self.current_elevation_position
+
+                    if (
+                        current_elevation_position is None
+                        or current_rotator_position is None
+                        or (
+                            (
+                                np.abs(rotation_angle - current_rotator_position)
+                                > self.rotation_angle_limit
+                            )
+                            or (
+                                np.abs(elevation - current_elevation_position)
+                                > self.elevation_angle_limit
+                            )
+                        )
+                    ):
+                        self.log.info(
+                            f"Current rotator position: {current_rotator_position:.1f}, "
```

Comment:
I just realized that this will fail if `current_elevation_position = None`. Sorry to drag this any more than it needs but you might want to do something like:

```
report_elevation = f"{current_elevation_position:.1f}" if current_elevation_position is not None else "not set"
``` 

Same for rotator I guess.